app/services/katello/pulp3/repository/apt.rb

```diff
@@ -203,9 +212,9 @@ def add_debs(source_repo_ids, filters, filter_list_map)
           filter_list_map
         end
 
-        def copy_content_from_mapping(repo_id_map, options = {})
+        def copy_content_from_mapping(repo_id_map, _options = {})
```

Comment:
I don't think `copy_content_from_mapping` will even run for Debian content, so these methods can likely be taken out entirely. If you trace back references to `copy_content_from_mapping`, you'll eventually get here: https://github.com/Katello/katello/blob/07d11cd10d3edbab9f1f73c66494b3ff1b53ca04/app/lib/actions/katello/content_view/publish.rb#L62

That shows that we only do the whole "multi copy" workflow for yum content anyway.

This wasn't always the case, but it has been since Katello 4.3.

Reply:
Although, if dep solving is something that is a TODO for debian content, then I'd recommend keeping these methods of course.

Reply:
I believe we will need it eventually for the IncrementalUpdate to work : https://github.com/Katello/katello/pull/10847

Dependency Solving for debian is currently out-of-scope and due to various reasons like lacking support for efficient handling (sorting and comparing) of debian version within PostgreSQL.
This is also the reason I added a PR to make sure dep-solving with debs is never even tried :smiling_imp: : https://github.com/Katello/katello/pull/10734